python/ray/experimental/state/api.py

```diff
@@ -50,6 +50,26 @@
         data = list_nodes(address="localhost:8265")
     ```
 """
+import logging
+
+logger = logging.getLogger(__name__)
+STATE_OBS_ALPHA_FEEDBACK_MSG = [
+    "\n==========ALPHA PREVIEW, FEEDBACK NEEDED ===============",
+    "State Observability APIs is currently in Alpha-Preview. ",
+    "If you have any feedback, you could do so at either way as below:",
+    "  1. Comment on API specification: https://tinyurl.com/api-spec",
+    "  2. Report bugs/issues with details: https://forms.gle/gh77mwjEskjhN8G46",
+    "  3. Follow up in #proj-state-obs-dogfooding slack channel.",
+    "==========================================================",
+]
+
+
+def _log_feedback_prompt(func):
```

Comment:
maybe we should only show them with `--help` option.

Reply:
Yeah, I was thinking of that - but this will then leave out the python SDK usage, e.g. when using jupyter notebooks.